jwst/resample/gwcs_drizzle.py

```diff
@@ -196,13 +196,10 @@ def add_image(self, insci, inwcs, inwht=None,
         wt_scl = 1.0  # hard-coded for JWST count-rate data
         self.increment_id()
 
-        dodrizzle(insci, inwcs, inwht, self.outwcs,
-                            self.outsci, self.outwht, self.outcon,
-                            expin, in_units, wt_scl,
-                            pscale_ratio=pscale_ratio, uniqid=self.uniqid,
-                            xmin=xmin, xmax=xmax, ymin=ymin, ymax=ymax,
-                            pixfrac=self.pixfrac, kernel=self.kernel,
-                            fillval=self.fillval)
+        dodrizzle(insci, inwcs, inwht, self.outwcs, self.outsci, self.outwht,
+                  self.outcon, expin, in_units, wt_scl, uniqid=self.uniqid,
+                  xmin=xmin, xmax=xmax, ymin=ymin, ymax=ymax,
+                  pixfrac=self.pixfrac, kernel=self.kernel, fillval=self.fillval)
```

Comment:
It's not clear to me why `dodrizzle` actually ever needs to pass `pscale_ratio` to the underlying `tdriz` C function.  All scaling is taken care of by the input and output frame WCS objects.  So I've removed it from the calling signature here and down at the call to `tdriz`.

Reply:
It might be worth running this by either @mcara or @stsci-hack to confirm.

Reply:
The parameter `pscale_ratio` is not necessary at this point, as the full transformation from input to output WCS has already been determined before calling this code when the `self.outwcs` WCS was defined/provided by the user.  So, this actually cleans up the code.